horusec-cli/internal/services/formatters/javascript/eslint/config.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eslint
+
+const (
+	ImageName = "horuszup/eslint"
+	ImageTag  = "latest"
```

Comment:
Change to use number equals in `deployments/dockerfiles/eslint/.semver.yaml` in your case `v0.0.1`

Reply:
this change is necessary to not broken when run script for update tools in horusec-cli

Reply:
you are right!